example/EmbeddedChessPieces/src/ECPChessLogic/ECPChessPiece.h

```diff
@@ -13,23 +13,52 @@
 #define ECPChessPiece_h
 
 #include <Dezibot.h>
+
 #include "ECPChessField.h"
+#include "ECPMovement/ECPMovement.h"
+
+
+#define COLOR_DELAY 2000
 
 /**
  * @brief Abstract class for chess piece, e.g. pawn, tower etc.
  * 
  */
 class ECPChessPiece {
 public:
-    const Dezibot& dezibot;
+    Dezibot& dezibot;
 
     /**
      * @brief Construct a new chess piece object
      * 
      * @param d Dezibot that simulates the piece
+     * @param ecpMovement Movement object of dezibot
      * @param initialField Initial field of chess piece on board, i.e. A1 white tower
```

Comment:
```suggestion
     * @param initialField Initial field of chess piece on board
```
Please remove examples consistently.

Reply:
Fixed in 2415e66